docker-compose-travis.yml

```diff
@@ -0,0 +1,17 @@
+# This is only currently used by Travis for testing
```

Comment:
This file should eventually be moved into the `.travis` directory.  See what I did in the ecommerce PR: https://github.com/edx/ecommerce/pull/1026#discussion_r89206094

When you do this you have to update the volume path.